.github/workflows/common-commitlint.yml

```diff
@@ -14,6 +20,7 @@ jobs:
         uses: extenda/actions/commitlint@v0
         with:
           message: ${{ github.event.pull_request.title }}
+          enable_pr_title_validation: ${{ inputs.enable_pr_title_validation }}
```

Comment:
Has this been pushed to the actions repo already? Our actions use kebab-case input variables.